web/src/message_edit.js

```diff
@@ -1332,3 +1332,31 @@ export function with_first_message_id(stream_id, topic_name, success_cb, error_c
         error: error_cb,
     });
 }
+
+export function is_message_oldest_or_newest(
+    stream_id,
+    topic_name,
+    message_id,
+    success_cb,
+    error_cb,
```

Comment:
We don't use abbreviations in the codebase as a code style thing; call this `success/error` or `success_callback/error_callback` as appropriate.

Reply:
Renamed to `success_callback/error_callback`, and I'll remember to use lesser abbreviations. Thanks!